src/legacy/core_plugins/kibana/index.js

```diff
@@ -176,7 +176,6 @@ export default function (kibana) {
               visualize: true,
               console: true,
               advanced_settings: true,
-              saved_objects: true,
```

Comment:
"saved_objects" in the catalogue aren't driven by ui capabilities at the moment.